src/main/java/com/imgix/URLHelper.java

```diff
@@ -64,8 +64,14 @@ public void deleteParameter(String key) {
 	}
 
 	private String encodeBase64(String str) {
-		byte[] stringBytes = str.getBytes();
-		String b64EncodedString = new String(Base64.getEncoder().encode(stringBytes));
+		byte[] stringBytes = new byte[0];
```

Comment:
just declare this variable inside the `try` block, or inline the variable. No point in allocating an array of length zero and then immediately throwing it away on line 71.